docs/src/main/sphinx/connector/hive.rst

```diff
@@ -99,6 +99,68 @@ Materialized views
 The Hive connector supports reading from Hive materialized views.
 In Trino, these views are presented as regular, read-only tables.
 
+Hive views
+----------
+
+Hive views are defined in Hive QL and stored in the Hive Metastore Service. They
+analyzed to allow read access to the data.
+
+The Hive connector includes support for reading Hive views with three different
+modes.
+
+* disabled
+* legacy
+* experimental
+
+You can configure the behavior in your catalog properties file.
+
+**disabled**
+
+The default behavior is to ignore Hive views. This means that your business
+logic and data encoded in the views is not available in Trino.
+
+**legacy**
+
+A very simple implementation to execute Hive views, and therefore allow read
+access to the data in Trino, can be enabled with
+``hive.translate-hive-views=true`` and
+``hive.legacy-hive-view-translation=true``.
+
+This legacy behavior interprets any Hive QL query that defines a view as if it
```

Comment:
May be just use `HiveQL` as a noun?

Reply:
okay.

Reply:
I noticed "Hive QL " and "HiveQL" being used in the doc.
https://en.wikipedia.org/wiki/Apache_Hive#HiveQL and https://docs.microsoft.com/en-us/azure/hdinsight/hadoop/hdinsight-use-hive suggests "HiveQL" to be preferred wording.

Reply:
I updated them all now to use HiveQL